.circleci/test-deploy.yml

```diff
@@ -1,38 +1,47 @@
 version: 2.1
 orbs:
-  <orb-name>: <namespace>/<orb-name>@dev:<<pipeline.git.revision>>
+  testlio: testlio/testlio@dev:<<pipeline.git.revision>>
   orb-tools: circleci/orb-tools@11.1
 
 filters: &filters
   tags:
     only: /.*/
 
 jobs:
-  # Create a job to test the commands of your orbs.
-  # You may want to add additional validation steps to ensure the commands are working as expected.
-  command-tests:
+  unzip-sample-test-project:
     docker:
-      - image: cimg/base:current
+      - image: cimg/node:lts
     steps:
       - checkout
-      # Run your orb's commands to validate them.
-      - <orb-name>/greet
+      - run:
+          name: Unzip Sample Test Project
+          command: "unzip testlio-platform-automation-main.zip"
+      - persist_to_workspace:
+          root: ./testlio-platform-automation-main
+          paths:
+            - ./*
 workflows:
   test-deploy:
     jobs:
-      # Make sure to include "filters: *filters" in every test job you want to run as part of your deployment.
-      - command-tests:
+      - unzip-sample-test-project:
           filters: *filters
+      - testlio/schedule-run:
+          filters: *filters
+          requires:
+            - unzip-sample-test-project
+          attach-workspace: true
+          token: "RUN_API_TOKEN_STAGING"
       - orb-tools/pack:
           filters: *filters
       - orb-tools/publish:
-          orb-name: <namespace>/<orb-name>
+          orb-name: testlio/testlio
           vcs-type: << pipeline.project.type >>
           pub-type: production
           requires:
             - orb-tools/pack
-            - command-tests
-          context: <publishing-context>
+            - unzip-sample-test-project
+            - testlio/schedule-run
```

Comment:
Do we need to create a new automated run for real to test the command? Could we use some sort of "[dry run](https://en.wikipedia.org/wiki/Dry_run_(testing))" instead?

The run creation is the job of the CLI, the orb job is simply to trigger the Testlio CLI command. Therefore, by creating an automated run for real we are actually testing the Testlio CLI instead of the Orb itself.

Reply:
Created the follow up ticket: https://testlions.atlassian.net/browse/GREEN-643